decision/engine.go

```diff
@@ -231,6 +240,8 @@ func (e *Engine) MessageReceived(p peer.ID, m bsmsg.BitSwapMessage) error {
 		l.wantList = wl.New()
 	}
 
+	blockSize, msgSize := 0, 0
```

Comment:
cleaner to write:
```
var blockSize, msgSize int
```

Reply:
@whyrusleeping I agree.
This is done. I pushed out these changes and squashed into a single commit.